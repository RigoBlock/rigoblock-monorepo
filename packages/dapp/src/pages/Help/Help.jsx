import React, { Component } from 'react'
import Navbar from '../../components/organisms/Navbar'

class Help extends Component {
  render() {
    return (
      <div className="Help">
        <Navbar />
        <h1>Help</h1>
      </div>
    )
  }
}

export default Help
