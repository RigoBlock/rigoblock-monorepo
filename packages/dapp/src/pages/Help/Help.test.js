import { shallow } from 'enzyme'
import Help from './Help.jsx'
import React from 'react'
import toJson from 'enzyme-to-json'

describe('Help page', () => {
  it('renders correctly', () => {
    expect(toJson(shallow(<Help />))).toMatchSnapshot()
  })
})
