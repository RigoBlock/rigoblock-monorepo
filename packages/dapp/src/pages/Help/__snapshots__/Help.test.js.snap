// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help page renders correctly 1`] = `
<div
  className="Help"
>
  <Connect(BaseTemplate)>
    <h1>
      Help
    </h1>
  </Connect(BaseTemplate)>
</div>
`;
